shared/src/business/entities/EntityConstants.ts

```diff
@@ -413,6 +413,14 @@ export const INTERNAL_DOCUMENTS_ARRAY = flatten(
   Object.values(DOCUMENT_INTERNAL_CATEGORIES_MAP),
 );
 
+export const MOTION_EVENT_CODES = [
+  ...DOCUMENT_INTERNAL_CATEGORIES_MAP['Motion'].map(entry => {
```

Comment:
During the amicus brief story, we discovered that internal and external filing events motion categories are different (one may be a subset of the other, I'm honestly not sure). Is there any concern that using just the internal list here will miss any e-filed motions?

Reply:
Seems like a good thing to check. I did notice that internal contained the generic "Motion" document type, M000, while external didn't.

Reply:
Okay, I got the difference between internal and external motions, and it looks like internal has an another 9 motion types: ['M000', 'M031', 'M032', 'M048', 'M057', 'M079', 'M082', 'M086', 'M123']. Do you think it seems valid to use internal then since it has more types? All 128 motion types for external exist within internal.

Reply:
Thanks for doing that investigation, sounds like it's good to merge then